server/server_listen.go

```diff
@@ -13,9 +15,10 @@ import (
 
 //TLSConfig enables configures TLS
 type TLSConfig struct {
-	Key     string
-	Cert    string
-	Domains []string
+	Key       string
+	Cert      string
+	Domains   []string
+	MtlsCaDir string
```

Comment:
Please rename this to `CA`

Reply:
Sure, as long as developers not confuse this CA with server cert CA.